src/core/core/events.js

```diff
@@ -0,0 +1,34 @@
+import EventEmitter from 'eventemitter2'
```

Comment:
As we discussed, having a folder called core inside core isn't very meaningful, shall we handle this as part of the story in which we remove the unused code?